packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -677,6 +732,15 @@ function useMediaUploadQueue() {
       (item) => item.state === ITEM_STATUS.MUTING
     );
 
+    /**
+     * Whether any video cropping is currently in progress.
+     *
+     * @type {boolean} Whether we're muting.
+     */
+    const isCropping = state.queue.some(
+      (item) => item.state === ITEM_STATUS.CROPPING
+    );
+
```

Comment:
Need to add ITEM_STATUS.CROPPING to `isCurrentResourceProcessing` and `isNewResourceProcessing`